src/fs/fs.c

```diff
@@ -40,6 +40,80 @@ LIST_HEAD(list, cfs_fs_node) fs_list = LIST_HEAD_INITIALIZER();
 /* global endpoint operations for cortxfs*/
 static const struct cfs_endpoint_ops *g_e_ops;
 
+static int fs_node_init(struct cfs_fs_node *fs_node,
+			struct namespace *ns, size_t ns_size)
+{
+	int rc = 0;
+
+	str256_t *fs_name = NULL;
+	ns_get_name(ns, &fs_name);
+
+	fs_node->cfs_fs.ns = calloc(1, ns_size);
+	if (fs_node->cfs_fs.ns == NULL) {
+		goto out;
+	}
+
+	memcpy(fs_node->cfs_fs.ns, ns, ns_size);
+
+	fs_node->cfs_fs.kvtree = calloc(1, sizeof(struct kvtree));
+	if (!fs_node->cfs_fs.kvtree) {
+		rc = -ENOMEM;
+		goto kvtree_alloc_fail;
+	}
+
+	rc = kvtree_init(fs_node->cfs_fs.ns, fs_node->cfs_fs.kvtree);
+	if (rc != 0) {
+		log_err("failed to load FS: " STR256_F
+			" , kvtree_init() failed!",
+			STR256_P(fs_name));
+		goto kvtree_init_fail;
+	}
+
+	fs_node->cfs_fs.root_node = malloc(sizeof(struct kvnode));
```

Comment:
Any reason of using malloc here and calloc above at line 58?

Reply:
Replaced malloc with calloc. I